### ISSUE TITLE
none is not an allowed value (type=type_error.none.not_allowed)

### ISSUE THREAD

**Issue by jnss1980** [continuedev/continue#523]
### Before submitting your bug report

- [ ] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [ ] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: MacBook Air Ventura 13.6
- Continue: 0.0.5
- IDE: IntelliJ IDEA 2023.2.1(Ultimate Edition)
- USE LlamaCpp Call API
```

### Description

Use IntelliJ IDEA to open a javascript file, click Method Name and press Ctrl +C to copy. This error message will pop up.

### To reproduce

Highlight code section , This error message will pop up.
<img width="835" alt="image" src="https://github.com/continuedev/continue/assets/8285263/42eab47b-e642-4eab-bf79-ead85ae515f6">

### Log output

```Shell
[2023-09-30 00:41:19,568] [CRITICAL] Exception caught from async task: Traceback (most recent call last):

  File "continuedev/src/continuedev/libs/util/create_async_task.py", line 21, in callback
    future.result()

  File "asyncio/futures.py", line 201, in result

  File "asyncio/tasks.py", line 232, in __step

  File "continuedev/src/continuedev/core/autopilot.py", line 288, in handle_highlighted_code
    await self.context_manager.context_providers["code"].handle_highlighted_code(

  File "continuedev/src/continuedev/plugins/context_providers/highlighted_code.py", line 244, in handle_highlighted_code
    item=self._rif_to_context_item(

  File "continuedev/src/continuedev/plugins/context_providers/highlighted_code.py", line 169, in _rif_to_context_item
    return ContextItem(

  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__

pydantic.error_wrappers.ValidationError: 1 validation error for ContextItem
editing
  none is not an allowed value (type=type_error.none.not_allowed)
```

**Comment by sestinj** [comment]
Hi @jnss1980 - JetBrains is still very much in progress. I wouldn't be surprised if there were a few more bugs to be found as well, but I'm working currently on bringing the extension to be on par with VS Code. I'll update you when we have a new version released to the store and this should then be solved!

**Comment by sestinj** [comment]
@jnss1980 the JetBrains extension is making good progress. If you'd like to do some more early testing, here's a pre-release that you can install from disk (it solves this above error and many others): https://discord.com/channels/1108621136150929458/1108621136830398496/1159049720799449119

We'd love to hear early feedback, but I'll also let you know again when we make the official release.

**Comment by sestinj** [comment]
@jnss1980 Another update: we've released version 0.0.8 to the JetBrains store and would love to hear your feedback as we work toward an official release next week!

I'm going to close this particular issue as I've verified that the problem was solved, but please open new ones if you get the chance to check out the new version : )
